lib/quill_delta.dart

```diff
@@ -800,3 +800,22 @@ class DeltaIterator {
     return op;
   }
 }
+
+extension DeltaUnsafe on Delta {
+  /// Creates new [Delta] from list of [operations]. Caller must ensure that
+  /// operations are valid inserts.
+  static Delta fromOperationsUnsafe(List<Operation> operations) {
+    assert(operations.every((op) => op.isInsert));
```

Comment:
Probably should not have the `insert` requirement, other operations are also valid.

```suggestion
  /// Creates new [Delta] from list of [operations].
  static Delta fromOperationsUnsafe(List<Operation> operations) {
```

Reply:
No argument here.